build.cmd

```diff
@@ -22,49 +21,54 @@ if /I "%1" == "/no-multi-proc" set MSBuildAdditionalArguments=&&shift&& goto :Pa
 call :Usage && exit /b 1
 :DoneParsing
 
-if not exist "%DeveloperCommandPrompt%" (
-  echo In order to build this repository, you need Visual Studio "15" Preview installed.
+if not exist "%VSINSTALLDIR%" (
```

Comment:
Looks like VS150COMNTOOLS is being set in jenkins and VSO - I'll respect that instead.